api-gateway/api-gateway-kubernetes.yml

```diff
@@ -23,6 +23,11 @@ spring:
             - Method=POST
           filters:
             - AddPrincipalToPathFilter
+        - id: social-media-adapter-api-doc
```

Comment:
I would better add one predicate to the existing service route, instead of creating a new route

Reply:
This is because the other only accepts the post http method.